pages/dao/[symbol]/params/components/ParamsView.tsx

```diff
@@ -137,7 +141,9 @@ const ParamsView = ({ activeGovernance, openGovernanceProposalModal }) => {
               tooltipMessage={
                 'Please connect wallet with enough voting power to create governance config proposals'
               }
-              onClick={openGovernanceProposalModal}
```

Comment:
This button opens the `GovernanceConfigModal` which edits governance config, not realm config. But you are replacing it with a link to the realm config form you made. So that seems like a bug.